navigation/antworld_rotater.h

```diff
@@ -52,14 +53,27 @@ class AntWorldRotater
         }
     }
 
-    size_t max() const
+    size_t numRotations() const
```

Comment:
In this case we do want it rounded down, because that's the number of rotations that will be carried out.

Come to think of it though, if you pick a value of ``m_YawStep`` that *doesn't* divide neatly into 360 deg then you've done something weird. Maybe we should enforce this with a ``BOB_ASSERT``?